_integration-tests/tests/fiber.v2/validation.json

```diff
@@ -0,0 +1,13 @@
+{
+  "url": "http://localhost:8089/ping",
+  "output": [
+    {
+      "name": "http.request",
+      "service": "fiber",
+      "resource": "GET /ping",
+      "type": "web",
+      "meta": {
+        "http.url": "/ping"
```

Comment:
This appears to be a potential regression from the [underlying middleware provider ](https://github.com/DataDog/dd-trace-go/blob/v1.58.0/contrib/gofiber/fiber.v2/fiber.go#L44) ... the other frameworks return the entire url while fiber returns only the path